project/build.scala

```diff
@@ -32,7 +32,7 @@ object build extends Build {
     tokens,
     transversers,
     trees
-  )
+  ) dependsOn (scalameta)
```

Comment:
yeah that might be better, but I'm not sure you can do `in scalameta` can you?

You could just set up an alias command.